modules/mono/editor/GodotTools/GodotTools/Export/ExportPlugin.cs

```diff
@@ -155,178 +103,81 @@ private void _ExportBeginImpl(string[] features, bool isDebug, string path, int
             if (!DeterminePlatformFromFeatures(features, out string platform))
                 throw new NotSupportedException("Target platform not supported");
 
+            if (!new[] { OS.Platforms.Windows, OS.Platforms.LinuxBSD, OS.Platforms.MacOS, OS.Platforms.Server }
+                    .Contains(platform))
+            {
+                throw new NotImplementedException("Target platform not yet implemented");
+            }
+
             string outputDir = new FileInfo(path).Directory?.FullName ??
-                               throw new FileNotFoundException("Base directory not found");
+                               throw new FileNotFoundException("Output base directory not found");
 
             string buildConfig = isDebug ? "ExportDebug" : "ExportRelease";
 
-            if (!BuildManager.BuildProjectBlocking(buildConfig, platform: platform))
-                throw new Exception("Failed to build project");
-
-            // Add dependency assemblies
+            // TODO: This works for now, as we only implemented support for x86 family desktop so far, but it needs to be fixed
+            string arch = features.Contains("64") ? "x86_64" : "x86";
 
-            var assemblies = new Godot.Collections.Dictionary<string, string>();
+            string ridOS = DetermineRuntimeIdentifierOS(platform);
```

Comment:
Currently exporting does not work:

1. The `DeterminePlatformFromFeatures` method fails to determine the platform since (at least in Linux) the relevant feature I get is "X11" which is not in the `PlatformNameMap` dictionary. It was renamed to "linux" in `master` (https://github.com/godotengine/godot/pull/47391) but keep in mind that the dictionary keys use the **OS.Names** constants so the closest thing would be "Linux" which is capitalized so it still wouldn't work.
2. If the previous method were to succeed the `platform` variable will be assigned the value "linuxbsd" which is passed to the `DetermineRuntimeIdentifierOS` and fails to determine the runtime id because "linuxbsd" is not in the `DotNetOSPlatformMap` dictionary (since the dictionary keys use the **OS.Names** constants, not the **OS.Platforms** constants).

Reply:
Yes, I noticed a few other issues as well. Turns out I had fixed those in later commits and should have included them.